### ISSUE TITLE
Specific banners for specific journal days

### ISSUE THREAD

**Issue by astubbs** [sawhney17/logseq-banners-plugin#66]
Is it possible to have a specific banner for a journal day? Or background? 

I can get it to work for a page, but using the same code, putting it on a journal day, doesn't seem to do anything - I just see the default one.

**Comment by ireina7** [comment]
Yes you can. Just check the plugin's `settings` section. There are options for `journal` and `common page` respectively.

**Comment by ireina7** [comment]
Maybe you should update your plugin? My plugin `logseq-banners-plugin` setting section does have options for journal banner:
![image](https://user-images.githubusercontent.com/73305492/211782762-22cca7d6-4f3b-4b8f-84ae-ad162c687c96.png)

**Comment by astubbs** [comment]
I just installed in last week, when I discovered Logseq.

Yes, there is a section in the settings for journal and home settings. 

But it only has a `default` banner setting. I'm not asking about settings a _default banner_ for all journal pages, I'm asking about _setting a specific banner_, for _a specific journal page_. I.e. :

`journals/2023_01_08.md`:
```
banner:: https://images.hdqwalls.com/download/ryu-hadouken-street-fighter-2-55-2048x1152.jpg

- LATER Fix banner entry
```

But here's what I see - just the default banner.:
<img width="1428" alt="Screenshot 2023-01-11 at 11 05 31 AM" src="https://user-images.githubusercontent.com/3414/211790742-76b1f110-752c-4043-b426-3849e4b2a1cd.png">

On a logseq `page`, doing the same thing above, works as expected:

<img width="1437" alt="Screenshot 2023-01-11 at 11 06 06 AM" src="https://user-images.githubusercontent.com/3414/211790854-a8c0d0dc-2f38-4b49-b067-55736981d838.png">


Here's my settings screenshot:
<img width="656" alt="Screenshot 2023-01-11 at 11 04 55 AM" src="https://user-images.githubusercontent.com/3414/211790632-eb0b004c-617a-4efd-be0a-7b86024e1925.png">

I hope that clarifies the question?

**Comment by ireina7** [comment]
Thanks for your clarification. Now I understand your issue.
You are right. I tried to set banner for specific journal page and the property just doesn't work.

**Comment by astubbs** [comment]
Cool :) Next question - is this a `bug`, or a `feature request`? 🤗

**Comment by ireina7** [comment]
I'm not quite sure since I currently don't have much time to debug this. But I guess this is a `feature request` after check some functions in `src` code. Maybe the rendering procedure uses the global config when rendering journal pages.

**Comment by astubbs** [comment]
Ok cool. I might take a look at it this weekend - really want to get that street fighter themed day ;)

**Comment by astubbs** [comment]
Oh - @ireina7 given that I might try to take a look at it - any hints you might give as to where/ what it might be would be useful :)

**Comment by HlexNC** [comment]
@astubbs Have you by any chance fixed the problem? I am trying to do the same thing, but for pages

**Comment by astubbs** [comment]
@HlexNC no, haven't looked into it yet and probably won't. 
But, banner image for a page works as expected for me. What problem with pages are you seeing?
See my example above - "On a logseq page, doing the same thing above, works as expected:"

**Comment by HlexNC** [comment]
> @HlexNC no, haven't looked into it yet and probably won't. But, banner image for a page works as expected for me. What problem with pages are you seeing? See my example above - "On a logseq page, doing the same thing above, works as expected:"

No problem... just being dumb. I didn't see `banner:: https://` in the README.md file.

**Comment by HlexNC** [comment]
![image](https://user-images.githubusercontent.com/66517969/214716131-50a78f4e-4720-4a6a-854c-9a0ab494f6d4.png){width 500}
Works well 😆

**Comment by stdword** [comment]
Guys, this is feature request. There is no handling a case for looking at the `banner::` prop for journal pages in the source code.

**Comment by blackwint3r** [comment]
Any progress of this? Really need this feature, want to add "the picture of my day" to my journal's banner, to enhance my autobiographical memory.
